react/components/ContainerStateSelector.tsx

```diff
@@ -72,16 +90,18 @@ const ContainerStateSelector: FC<Props> = ({ showSelectSkuMessage, googleMapsKey
   }
 
   return (
-    <div className="flex flex-column">
-      <div className="c-muted-2 t-body mb1 ml3">
+    <div className={`${handles.innerContainer} flex flex-column`}>
+      <div className={`${handles.availabilityHeader} c-muted-2 t-body mb1 ml3`}>
         <FormattedMessage id="store/pickup-availability.availability-header" />
       </div>
-      <div className="c-on-base t-body mb1 ml3 mt3">
+      <div
+        className={`${handles.availabilityMessage} c-on-base t-body mb1 ml3 mt3`}
+      >
         <FormattedMessage id="store/pickup-availability.available-for-pickup" />
       </div>
-      <div>
+      <div className={`${handles.availabilityButtonContainer}`}>
```

Comment:
I'm proposing this change so if we add another button we don't get stuck

```suggestion
      <div className={`${handles.availabilityChooseStoreButton}`}>
```

Reply:
neaaat